### PR TITLE
added bounty hunter program doc

### PR DESCRIPTION
**Pull request by hiccups28**
still need to do the formatting

### CODE REVIEW

File: docs/lakeFSBountyHunterProgram.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+lakeFS Bounty Hunter Program
```
Comment (adipolak) [inline docs/lakeFSBountyHunterProgram.md:1]: this is missing a header that connects all our new pages together. 
@hiccups28 take a look at this one as an example - https://raw.githubusercontent.com/treeverse/lakeFS/master/docs/contributing.md

**Review by adipolak — CHANGES_REQUESTED** [review]

File: docs/lakeFSBountyHunterProgram.md (hunk 1 above)
Comment (nopcoder) [inline docs/lakeFSBountyHunterProgram.md:1]: These programs called "Bug Bounty" without the hunter

File: docs/lakeFSBountyHunterProgram.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+lakeFS Bounty Hunter Program is a community program to encourage lakeFS Active OSS users to engage with the source code and improve lakeFS capabilities. The program is designed to reward community members who actively find bugs and suggest product improvements within lakeFS. Through this program, we hope to empower and recognize veteran lakeFS users.
```
Comment (johnnyaug) [inline docs/lakeFSBountyHunterProgram.md:3]: No need to capitalize "Active" throughout this doc.

File: docs/lakeFSBountyHunterProgram.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+lakeFS Bounty Hunter Program
+
+lakeFS Bounty Hunter Program is a community program to encourage lakeFS Active OSS users to engage with the source code and improve lakeFS capabilities. The program is designed to reward community members who actively find bugs and suggest product improvements within lakeFS. Through this program, we hope to empower and recognize veteran lakeFS users. 
+
+*Video going through the bounty hunter program*
+
+Getting Started
```
Comment (johnnyaug) [inline docs/lakeFSBountyHunterProgram.md:7]: Please use markdown headings.

File: docs/lakeFSBountyHunterProgram.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Before we get started, please see the requirements to join the program:
```
Comment (johnnyaug) [inline docs/lakeFSBountyHunterProgram.md:9]:
```suggestion
### Requirements
```

File: docs/lakeFSBountyHunterProgram.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+New Bounty Users must have an Active lakeFS OSS installation 
+New Users must be an active OSS users for at least 1 month
+Must be a member of lakeFS Community Slack
+To be an active OSS users, must be apart of 6 versions,
```
Comment (johnnyaug) [inline docs/lakeFSBountyHunterProgram.md:14]: What?

File: docs/lakeFSBountyHunterProgram.md [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+
+*Video going through the bounty hunter program*
+
+Getting Started
+
+Before we get started, please see the requirements to join the program:
+
+New Bounty Users must have an Active lakeFS OSS installation 
+New Users must be an active OSS users for at least 1 month
+Must be a member of lakeFS Community Slack
```
Comment (johnnyaug) [inline docs/lakeFSBountyHunterProgram.md:13]:
```suggestion
- To join the program, the user must have a lakeFS installation that has been active for at least 1 month.
- The user must be a member of the lakeFS community Slack.
```

File: docs/lakeFSBountyHunterProgram.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+To become a new lakeFS Bounty Hunter, please apply here.
+
+First & last name
+Github Username
+Email
+Company Name & Position
+Do you use lakeFS for work?
+lakeFS Active OSS installation 
+
+New Users will receive an email to verify that they have been accepted to the Bounty Hunter Program and will be given a link to access the bounties.
```
Comment (johnnyaug) [inline docs/lakeFSBountyHunterProgram.md:25]:
```suggestion
New Users will receive an email to confirm that they have been accepted to the Bounty Hunter Program and will be given a link to access the bounties.
```

File: docs/lakeFSBountyHunterProgram.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+New Issues will be assigned to Bounty Users as a first come first serve basis. Bounty Users can also open new issues if they find any along the way. Issue cards will always contain the following information:
+
+lakeFS Version
+Type of Installation 
+Description
+Error Code
+How to reproduce the error
+
+Bounty Review Process
+All new issues submitted to the lakeFS bounty program will be reviewed by a lakeFS Developer. Here are the list of submission acceptable for review:
```
Comment (johnnyaug) [inline docs/lakeFSBountyHunterProgram.md:45]:
```suggestion
All new issues submitted to the lakeFS bounty program will be reviewed by a lakeFS developer. Here is the list of submission types acceptable for review:
```

**Review by johnnyaug — CHANGES_REQUESTED** [review]
Please rename the file to conform with the convention of lower_snake_case.

Is this going to be a page in the docs, or is this an internal document?

If it's going to be public, it still requires proofreading - there are a lot of typos. Also, missing metadata headers (see at the top of other doc page). Of course, also missing the videos, links and usage of markdown formatting.

**Comment by vinodhini-sd** [comment]
@adipolak Closing the PR as we decided to move forward with a web landing page instead of docs.

**Pull request closed without merging**